<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDDialog</class>
 <widget class="QWidget" name="BDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="timestampLabel">
     <property name="text">
      <string>Timestamp:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelID">
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Imagen</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="4" column="1">
    <widget class="QListWidget" name="listWidget_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
